TowerForge/TowerForge/ViewControllers/RegisterViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  RegisterViewController.swift
+//  TowerForge
+//
+//  Created by Vanessa Mae on 08/04/24.
+//
+
+import Foundation
+import UIKit
+
+class RegisterViewController: UIViewController {
+    @IBOutlet private var emailInput: UITextField!
+    @IBOutlet private var usernameInput: UITextField!
+    @IBOutlet private var passwordInput: UITextField!
```

Comment:
Not sure how exactly it is done in UIKit but should we maybe turn the password text field into like one with secure text input? Shouldn't be a major issue though.

Reply:
Yep will resolve this and make a github issue